modules/images-configuration-shortname.adoc

```diff
@@ -93,11 +93,40 @@ When the `allowedRegistries` parameter is defined, all registries, including the
 ifndef::openshift-rosa,openshift-dedicated[]
 .Verification
 
-* To check that the registries have been added, when a node returns to the `Ready` state, use the following command on the node:
+* To check that the registries have been added to the policy file, enter the following command to obtain a list of your nodes:
 +
 [source,terminal]
 ----
-$ cat /host/etc/containers/registries.conf.d/01-image-searchRegistries.conf
+$ oc get node
+----
++
+Example output
++
+[source,terminal]
```

Comment:
```suggestion
[source,text]
```

I've always preferred to make this a no-op. `terminal` sometimes highlights leading `$` or `#` because it's for terminal commands.